Dockerfile

```diff
@@ -17,14 +17,24 @@ RUN npm install
 
 
 #
-# Stage: Development environment
+# Stage: Basic Storybook
 #
-FROM node AS dev
-ENV NODE_ENV=development
+FROM node AS storybook
 
 COPY --from=npm /app/ .
+COPY .storybook/ .storybook/
+COPY src/ src/
+
+
+
+#
+# Stage: Development environment
+#
+FROM storybook AS dev
+ENV NODE_ENV=development
+EXPOSE 8080
 
-CMD ["sleep", "86400"]
+CMD ["npx", "start-storybook", "--port", "8080"]
 
 HEALTHCHECK --interval=5s --timeout=1s \
-    CMD node --version
+    CMD wget --quiet --tries=1 --spider http://localhost:8080/ || exit 1
```

Comment:
Better to use `nc` to avoid logs? (Since it's a in-browser application, shouldn't make any difference.)

Reply:
Logs shouldn't be a problem (besides the `--quiet`), though as long as the Node application is ready as soon as it starts to listen to the port doesn't make much difference (only if something is not set up correctly).

Now remembering what the `|| exit 1` is for, conforming to healthcheck standard exit codes.

Reply:
(Re logs, more thinking of `prod` where it'll be Nginx, so quite visible.)

Reply:
This healthcheck seems specific to `dev`, but the Nginx `prod` configuration can turn the log off for `/ping`|`/health` or similar.